### PR TITLE
Internal/generate bat for tests

### PR DESCRIPTION
**Pull request by MajoBerger**
| Phases            | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |    1 |     0 |      0 |        0 |
| Developing      |  0  |    1 |    0 |      0 |         0 |
| Review             |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -   |   -     |        0 |
## Problem description
Generate .bats for testing based on .yml configuration

### CODE REVIEW

**Review by milanmajchrak — APPROVED** [review]

File: scripts/envs/__basic.example.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-rem Set those paths to relevant places in your computer, copy this file and rename the copy to "paths.bat"
+rem Set those paths to relevant places in your computer, copy this file and rename the copy to "__basic.bat"
 set dspace_source=C:\workspace\DSpace\
 set tomcat=C:\apache-tomcat-9.0.64\
 set dspace_application=C:\dspace\
```
Comment (milanmajchrak) [inline scripts/envs/__basic.example.bat:4]: 'No endline at end of file'

File: scripts/regenerate.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+java -jar regenerate.jar
```
Comment (milanmajchrak) [inline scripts/regenerate.bat:1]: 'No endline at end of file'

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

File: scripts/envs/__basic.example.bat (hunk 1 above)
Comment (MajoBerger) [reply scripts/envs/__basic.example.bat:4]: is that problem?

File: scripts/regenerate.bat (hunk 2 above)
Comment (Paurikova2) [inline scripts/regenerate.bat:1]: is this necessary?

File: scripts/__tests.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+# source folder - part of dspace where tests should be run - required
+# class name - required, without .java
+# debug: true - optional, use for debug
+# comment - optional
+# methondName - optional -use if want to run only one method
+# when ready, click regenerate.bat
+# WARNING - all .bats not in this file will be removed
+
+tests:
+  - sourceFolder: "dspace-server-webapp"
```
Comment (Paurikova2) [inline scripts/__tests.yaml:12]: Add comment e.g. #example one

File: scripts/__tests.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# copy this file and name it tests.yaml
```
Comment (Paurikova2) [inline scripts/__tests.yaml:1]: Add here
"create your own test sets using these commands - not all of them are necessary"

**Review by Paurikova2 — CHANGES_REQUESTED** [review]
Generates only one file, if there are several tests of same class, one with method name, one without it.

**Review by Paurikova2 — APPROVED** [review]

**Pull request merged**